src/components/date-filter-v2/date-filter-v2.types.ts

```diff
@@ -0,0 +1,24 @@
+import { type Dayjs } from 'dayjs';
+
+import { type DATE_FILTER_RELATIVE_VALUES } from './date-filter-v2.constants';
+
+export type RelativeDurationConfig = {
+  label: string;
+  durationSeconds: number;
+};
+
+export type RelativeDateFilterValue = keyof typeof DATE_FILTER_RELATIVE_VALUES;
+
+export type DateFilterValue = Dayjs | 'now' | RelativeDateFilterValue;
```

Comment:
See if you can pass Dates here, and leave Dayjs as an internal implementation detail of the date picker